packages/jest-matchers/src/__tests__/spyMatchers-test.js

```diff
@@ -60,40 +60,37 @@ const jestExpect = require('../').expect;
   });
 });
 
-['toHaveBeenCalled', 'toBeCalled'].forEach(matcherName => {
-  test(`${matcherName} works only on spies or jest.fn`, () => {
-    let error;
-    const fn = () => {};
-
-    try {
-      jestExpect(fn)[matcherName]();
-    } catch (e) {
-      error = e;
-    }
-
-    expect(error).toBeDefined();
-    expect(error).toMatchSnapshot();
-  });
-});
-
-describe('.toHaveBeenCalledTimes()', () => {
+describe('toHaveBeenCalledTimes', () => {
   it('accepts only numbers', () => {
     const fn = jasmine.createSpy('fn');
     fn();
     jestExpect(fn).toHaveBeenCalledTimes(1);
 
     [{}, [], true, 'a', new Map(), () => {}].forEach(value => {
-      expect(() => jestExpect(fn).toHaveBeenCalledTimes(value))
-        .toThrowError(
-          /toHaveBeenCalledTimes expected value should be a number/);
+      let error;
+      try {
+        jestExpect(fn).toHaveBeenCalledTimes(value);
+      } catch (e) {
+        error = e;
+      }
+
+      expect(error).toBeDefined();
+      expect(error).toMatchSnapshot();
     });
   });
 
   it('verifies that actual is a Spy', () => {
     const fn = () => {};
 
-    expect(() => jestExpect(fn).toHaveBeenCalledTimes(2))
-      .toThrowError(/toHaveBeenCalledTimes matcher can only be used on a spy or mock function/);
+    let error;
```

Comment:
you can use this: https://github.com/facebook/jest/blob/master/packages/jest-matchers/src/__tests__/_matchErrorSnapshot.js